### PR TITLE
Use new helm upgrade --set format

### PR DESCRIPTION
**Pull request by ricekot**

### CODE REVIEW

**Comment by hnataraj** [comment]
@ricekot does the k8s quick start instructions need modifications?

**Comment by hnataraj** [comment]
@ricekot also do we need to change config management tasks in sensor?

File: docs/beta/api-observability/common-tasks/sensor/sensor-configuration.mdx [hunk 1]
```diff
@@ -137,17 +137,28 @@ Configuration is specified via a [Helm Values](https://helm.sh/docs/chart_templa
...
+# Specify the 'Application Name' chosen earlier in the config-values.yml file.
 #
-helm upgrade --install -n levoai --create-namespace \
+helm upgrade levoai-sensor levoai/levoai-ebpf-sensor \
+  --install \
+  --namespace levoai \
+  --create-namespace \
   --set sensor.otel.grpcEndpoint=<hostname|IP:port> \
-  --set "sensor.args={--default-service-name,<'Application Name' chosen earlier>}" \
-  levoai-sensor levoai/levoai-ebpf-sensor \
```
Comment (hnataraj) [inline docs/beta/api-observability/common-tasks/sensor/sensor-configuration.mdx:146]: @ricekot are these two lines not required?

--set "sensor.args={--default-service-name,<'Application Name' chosen earlier>}" \
  levoai-sensor levoai/levoai-ebpf-sensor \

Comment (ricekot) [reply docs/beta/api-observability/common-tasks/sensor/sensor-configuration.mdx:146]: The installation and image names have been moved to line 142, and the application name can be provided in the config-values.yml file (as suggested on line 140).

File: docs/beta/api-observability/install-guide/install-sensor.md [hunk 2]
```diff
@@ -43,13 +43,15 @@ helm repo add levoai https://charts.levo.ai && helm repo update
 # Replace 'hostname|IP' & 'port' with the values you noted down from the Satellite install
 # If Sensor is installed on same cluster as Satellite, use 'levoai-collector.levoai:4317'
 #
-# Specify below the 'Application Name' chosen earlier. Do not quote the 'Application Name'
-# Example: sensor.args={--default-service-name,my-test-app}
+# Specify below the 'Application Name' chosen earlier.
```
Comment (hnataraj) [inline docs/beta/api-observability/install-guide/install-sensor.md:46]: @ricekot what about line 46? What is it now referring to?

**Comment by hnataraj** [comment]
@ricekot LGTM

**Pull request merged**